test/lit/passes/coalesce-locals-gc.wast

```diff
@@ -149,20 +149,18 @@
   )
  )
 
- ;; CHECK:      (func $unreachable-get-null (type $4)
+ ;; CHECK:      (func $unreachable-get-null (type $3)
  ;; CHECK-NEXT:  (local $0 anyref)
  ;; CHECK-NEXT:  (local $1 i31ref)
  ;; CHECK-NEXT:  (unreachable)
  ;; CHECK-NEXT:  (drop
  ;; CHECK-NEXT:   (block (result anyref)
- ;; CHECK-NEXT:    (unreachable)
+ ;; CHECK-NEXT:    (ref.null none)
  ;; CHECK-NEXT:   )
  ;; CHECK-NEXT:  )
  ;; CHECK-NEXT:  (drop
  ;; CHECK-NEXT:   (block (result i31ref)
- ;; CHECK-NEXT:    (ref.i31
- ;; CHECK-NEXT:     (i32.const 0)
- ;; CHECK-NEXT:    )
+ ;; CHECK-NEXT:    (ref.null none)
```

Comment:
Comment on new line 168 needs to be updated. We do emit a ref.null now, but inside a block, so it's ok.